popup.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+// var limitMinutes = Infinity;
+var form = document.getElementById('block-form');
+
+
+// save given limit in local storage
+chrome.storage.sync.get(['limitInMiliseconds'], (results) => {
+    if (results.limitInMiliseconds != undefined && results.limitInMiliseconds != null) {
+        document.getElementById('block-after').value = results.limitInMiliseconds / 60 / 1000
+    }
+});
+
+
+// sets back to infity if you dont want to use it anymore
+document.getElementById('reset-btn').addEventListener('click', (event) => {
+    setNewLimit(Infinity)
+})
+
+form.addEventListener('submit', (event) => {
+    event.preventDefault(); // to prevent submit event to go to popup.html?limit=x
+    setNewLimit(document.getElementById('block-after').value * 60 * 100)
```

Comment:
value * 60 * 1000 yapsak?

Reply:
Ah pardon gozumden kacmis :+1:

Reply:
Desteğiniz için çok teşekkür ederim

Reply:
Ne demek hic sorun degil. [Chrome Api'ina ](https://developer.chrome.com/extensions/api_index) bakarsaniz aslinda daha kolay yontemler var redirection vs icin. Ben vikipediyi wikizeroya redirect eden kucuk bir [extension](https://github.com/aomerk/WikiZeroRedirector) yazmistim mesela ona bakabilirsiniz ornek olarak